### PR TITLE
new remote rename command

### LINKED ISSUE #1301: cmd ref: document new subcommand `remote rename`
Related to issue [iterative/dvc#3599](https://github.com/iterative/dvc/issues/3599).

### PR DESCRIPTION
**Pull request by karajan1001**
fix #1301 (match iterative/dvc/pull/3794)

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by karajan1001** [comment]
@jorgeorpinel
@PseudoNerd

Not a native English speaker, and never wrote front-end code. Actually, I can't do much more than now.

**Comment by jorgeorpinel** [comment]
This is perfect @karajan1001! We'll take it form here. Thank you so much.

File: content/docs/command-reference/remote/rename.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# remote rename
+
+Rename a data remote. This command affects DVC configuration files only, it does
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/rename.md:3]: Just adding a quick suggestion to check whether I can commit to this branch:
```suggestion
Rename a DVC remote. This command affects DVC configuration files only, it does
```

**Comment by jorgeorpinel** [comment]
- [ ] waiting for iterative/dvc/pull/3794 to merge

File: content/docs/command-reference/remote/rename.md (hunk 1 above)
Comment (karajan1001) [reply content/docs/command-reference/remote/rename.md]: Thank you, I would synchronize this to the DVC repo.

**Comment by efiop** [comment]
@jorgeorpinel That PR is merged now :slightly_smiling_face:

**Comment by shcheklein** [comment]
@jorgeorpinel are we ready to merge https://github.com/iterative/dvc.org/pull/1319 ? Any other concerns here?

**Comment by shcheklein** [comment]
@jorgeorpinel @karajan1001 

we should also update (or make sure that they are up to date):

- [ ] bash/zsh autocompletion scripts (DVC core repo)
- [ ] JS highlighters in this repo to add `dvc remote rename`

**Comment by karajan1001** [comment]
> @jorgeorpinel @karajan1001
> 
> we should also update (or make sure that they are up to date):
> 
> * [ ]  bash/zsh autocompletion scripts (DVC core repo)
> * [ ]  JS highlighters in this repo to add `dvc remote rename`

Thank you, I had opened a new issue #3847 in the core repo to do this.

**Comment by jorgeorpinel** [comment]
Hey guys I cherry-picked @karajan1001's commits into my branch for #1283 instead of pushing here.

I also fixed

>  JS highlighters in this repo to add dvc remote rename

and other general scaffolding stuff to put this doc in the right place. Closing this PR but thanks again!

**Comment by efiop** [comment]
> But may I ask a question? I had never seen an autocompletion on any DVC commands and didn't know their existence. How can I make them come into effect?

@karajan1001 Pip doesn't install those, unfortunately. You can install them manually using this guide https://dvc.org/doc/install/completion#shell-tab-completion . deb/rpm/brew packages automatically install it though.

**Comment by shcheklein** [comment]
@jorgeorpinel let's merge this and do improvements on top? It's not good that we don't attribute commits and contributions properly this way.

**Comment by efiop** [comment]
@shcheklein cherry-picking preserves the author of commits. But yeah, shame that this PR wasn't preserved. Though it is a common practice in many projects.

**Comment by shcheklein** [comment]
@efiop @jorgeorpinel yep, I didn't check that Jorge did a cherry-pick. I think it's fine then if it's easier to manage the process this way.

**Comment by jorgeorpinel** [comment]
Yes the commits are preserved. But you're right it's better for the contributor to have a merged PR. Reopening, we can merge it, the result will be the same in the end!

**Comment by jorgeorpinel** [comment]
However let's wait for mine to merge first? It already has all the improvements to this 👍 

Added a checkbox in https://github.com/iterative/dvc.org/pull/1283#issuecomment-633148473

**Comment by shcheklein** [comment]
@jorgeorpinel feel free to merge in any order :) I think both are approved?

**Comment by karajan1001** [comment]
@jorgeorpinel @efiop @shcheklein
Thank all of you. It's all OK. I am not familiar with Regular, and can't finish this by myself.

**Pull request merged**